### PR TITLE
Dev Portal - Updates to dashboard admin

### PR DESCRIPTION
**Pull request by ryanchristo**
## Description

This pull request updates the admin page in the dashboard to verify access using the colony admin role (same verification used in api.colony.io) and it adds a page showing a list of users. This pull request also includes some minor updates to package versions and adds `.env` examples.

**New stuff** ✨

* `.env` examples
* `useColonyRoles` hook

**Changes** 🏗

* Update colony and react package versions
* Update `SERVER_URL` fallback to local environment

### CODE REVIEW

File: src/modules/layouts/components/DeveloperPortalLayout/useColonyRoles.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    if (colonyClient && wallet) {
+      const { hasRole: hasRoleRoot } = await colonyClient.hasColonyRole.call({
+        address: wallet.address,
+        domainId: 1,
+        role: 'ROOT',
+      });
+      const { hasRole: hasRoleAdmin } = await colonyClient.hasColonyRole.call({
+        address: wallet.address,
+        domainId: 1,
+        role: 'ADMINISTRATION',
```
Comment (ceolson01) [inline src/modules/layouts/components/DeveloperPortalLayout/useColonyRoles.js]: `colony-js-client` has some constants for these 👍 

```jsx
import { COLONY_ROLE_ADMINISTRATION, COLONY_ROLE_ROOT } from '@colony/colony-js-client';
```

File: src/modules/pages/components/Dashboard/Admin/Admin.jsx [hunk 2]
```diff
@@ -43,11 +51,20 @@ type Props = {|
...
   path: string,
   user: User,
+  wallet: WalletObjectType,
 |};
 
-const Admin = ({ colonyClient, network, user }: Props) => {
-  const [visible, setVisible] = useState('AddAdmin');
-  if (!user || !user.admin || !user.admin[network.slug]) {
+const Admin = ({ colonyClient, network, user, wallet }: Props) => {
+  const [visible, setVisible] = useState('Accounts');
```
Comment (ceolson01) [inline src/modules/pages/components/Dashboard/Admin/Admin.jsx]: I know we didn't do this before, but I feel like we should flow type the state here. Something along the lines of:

```suggestion
  const [visible, setVisible] = useState<'Accounts' | 'AddAdmin'>('Accounts');
```

**Review by ceolson01 — APPROVED** [review]
LGTM! ✅ 

I left a couple notes for minor things, but nothing blocking. Nice work

File: src/modules/layouts/components/DeveloperPortalLayout/useColonyRoles.js (hunk 1 above)
Comment (ryanchristo) [reply src/modules/layouts/components/DeveloperPortalLayout/useColonyRoles.js]: Right! Good call.

File: src/modules/pages/components/Dashboard/Admin/Admin.jsx (hunk 2 above)
Comment (ryanchristo) [reply src/modules/pages/components/Dashboard/Admin/Admin.jsx]: Yup, I seemed to have gotten lazy. Updated. :+1:

**Pull request merged**